src/Config.ts

```diff
@@ -22,6 +22,44 @@ export class SocketOptions {
     //TO-DO
 }
 
+export interface SslOptions {
```

Comment:
I didn't realize that class's can not have optional field. So we want optional fields.